docs/apache-airflow-providers-amazon/operators/glacier.rst

```diff
@@ -15,17 +15,17 @@
     specific language governing permissions and limitations
     under the License.
 
-=================
-Amazon S3 Glacier
-=================
+===========
+AWS Glacier
```

Comment:
I'd keep S3 because the service is still S3. Glacier is only a storage class of S3

Reply:
Yep, I'd be incline with that

Reply:
Yeah, Glacier is an odd one.  Technically it's just part of S3 but they kind of pitch it like it's its own thing at times.